.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
+omit =
+    **/migrations/*
```

Comment:
You excluded only migrations. But what about configuration / system files (settings.py, setup.py, admin.py, apps.py, etc.)?  In accordance with my previous comment, these files must be excluded as well. Unit tests also must be excluded.